fpga/Makefile

```diff
@@ -120,9 +135,19 @@ debug-bitstream: $(build_dir)/obj/post_synth.dcp
 			$(build_dir)/debug_obj \
 			$(fpga_common_script_dir)
 
+MCS_FILE := $(build_dir)/obj/$(MODEL).mcs
```

Comment:
What is the MCS file used for? I removed this when originally setting up this work to keep things clean.

Reply:
Since every time we power off the board, the uploaded bit-stream will be lost. When working around the kernel after building up the prototype, we would not want to upload the bit-stream too often. Having a mcs file is better choice. It will save our time in the post stage of testing the prototype!